src/test/java/uk/gov/justice/probation/courtcaseservice/restclient/OffenderRestClientIntTest.java

```diff
@@ -84,4 +86,12 @@ public void whenGetConvictionRequirementsCalled_thenMakeRestCallToCommunityApi()
         assertThat(optionalRequirements.get()).hasSize(3);
     }
 
+    @Test
+    public void whenGetConvictionRequirementsCalled_thenMakeRestCallToCommunityApiEmptyRequirements() {
```

Comment:
This name doesn't really describe what the test is doing, I would suggest the following:
```suggestion
    public void givenEmptyRequirementsWillBeReturnedFromCommunityApi_whenGetConvictionRequirementsCalled_thenReturnEmptyRequirements() {
```

Reply:
This is what I've been going for: https://martinfowler.com/bliki/GivenWhenThen.html

Reply:
I copied the existing method and added what happens to the "then" part "whenGetConvictionRequirementsCalled_thenMakeRestCallToCommunityApi" which was already there. I'll change both methods ?